website/docs/usage/projects.md

```diff
@@ -148,6 +148,13 @@ skipped. You can also set `--force` to force re-running a command, or `--dry` to
 perform a "dry run" and see what would happen (without actually running the
 script).
 
+By default `spacy projects run` checks your installed dependencies to verify
+that your environment is properly set up and aligns with the project's
+`requirement.txt`, if there is one. If missing or conflicting dependencies are
+detected, a corresponding warning is displayed. If you'd like to disable the
+dependency check, set `check_requirements: False` in your project's
```

Comment:
Aren't the standard yaml values lowercase?

```suggestion
dependency check, set `check_requirements: false` in your project's
```

Reply:
I think [both should be fine](https://yaml.org/type/bool.html).